_data/repos.json

```diff
@@ -497,11 +497,11 @@
       "homepage": ""
     },
     {
-      "name": "Scap Security Guide (SSG)",
+      "name": "ComplianceAsCode Project",
       "html_url": "https://github.com/ComplianceAsCode/content",
-      "forks_count": 315,
-      "stargazers_count": 755,
-      "description": "Security compliance content in SCAP, Bash, Ansible, and other formats",
+      "forks_count": 368,
+      "stargazers_count": 887,
+      "description": "Security automation content in SCAP, OSCAL, Bash, Ansible, and other formats.",
```

Comment:
I believe this description is automatically pulled from the one in the repo at https://github.com/ComplianceAsCode/content

Changing the description here won't make sense if not updated there first, as this will automatically be replaced when pulling from the repo's own metadata.

Reply:
Updated!

Reply:
This matches now, except for the period at the end, so it may get clobbered by a script in future to remove that. Could remove it now to reduce the churn.

Reply:
Ah, ok. I thought ``description`` was meant to be a full sentence & needed an ending period. Will remove.

Reply:
I don't think it matters whether it's a full sentence or not. What matters here is that it's out of sync with what is in the GitHub repo and will be clobbered next time an automated script updates this file. :smiley_cat: